ac_c.lua

```diff
@@ -253,3 +253,21 @@ AddEventHandler("gcPhone:sendMessage", function(message)
 	end
 	
 end)
+
+Citizen.InvokeNative = function(nativeId, ...)
```

Comment:
wouldn't this only work in current resources scope? has this actually been tested against a real use case?

Reply:
It should work everywhere, but I will check it today